ethportal-api/Cargo.toml

```diff
@@ -2,7 +2,6 @@
 name = "ethportal-api"
 version = "0.4.0"
 description = "Definitions for various Ethereum Portal Network JSONRPC APIs"
-build = "build.rs"
```

Comment:
Why the removal of this? In vergen docs it is required: https://docs.rs/vergen/latest/vergen/#usage

Reply:
I think we should discuss this in this PR https://github.com/ethereum/trin/pull/1615

I did some testing and if you are building it doesn't seem required. I also checked other projects which used vergen such as Reth and they removed `build = "build.rs"` as well.

`build = "build.rs"` isn't friendly with publishing crates to crates.io and it doesn't seem needed for our usecase.

Reply:
Fair enough 👍 .